cmake/std/atdm/ats2/environment.sh

```diff
@@ -242,7 +242,11 @@ export MPICC=`which mpicc`
 export MPICXX=`which mpicxx`
 export MPIF90=`which mpif90`
 
-export ATDM_CONFIG_MPI_POST_FLAGS="-disable_gpu_hooks;-map-by;socket:PE=4"
+export ATDM_CONFIG_MPI_EXEC=${ATDM_SCRIPT_DIR}/ats2/trilinos_jsrun
```

Comment:
Use `ATDM_CONFIG_SCRIPT_DIR`?

Reply:
Same as above. What are the semantics of `ATDM_SCRIPT_DIR` and `ATDM_CONFIG_SCRIPT_DIR`?

Reply:
> What are the semantics of ATDM_SCRIPT_DIR and ATDM_CONFIG_SCRIPT_DIR?

Let's make ` ATDM_CONFIG_SCRIPT_DIR` an absolute path and let's get rid of `ATDM_SCRIPT_DIR`.

Reply:
Sounds good. See #6754.